<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="0" total="3" passed="2">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="205035" started-at="2022-09-12T00:08:55 IST" finished-at="2022-09-12T00:12:20 IST">
    <groups>
    </groups>
    <test name="Default test" duration-ms="205035" started-at="2022-09-12T00:08:55 IST" finished-at="2022-09-12T00:12:20 IST">
      <class name="listnerpractice.loginTest">
        <test-method status="PASS" signature="Aaunchbrowser()[pri:0, instance:listnerpractice.loginTest@491cc5c9]" name="Aaunchbrowser" duration-ms="176456" started-at="2022-09-12T00:08:55 IST" finished-at="2022-09-12T00:11:51 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Aaunchbrowser -->
        <test-method status="PASS" signature="Aaunchbrowser()[pri:0, instance:listnerpractice.loginTest@491cc5c9]" name="Aaunchbrowser" duration-ms="27645" started-at="2022-09-12T00:11:51 IST" finished-at="2022-09-12T00:12:19 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Aaunchbrowser -->
        <test-method status="FAIL" signature="Browser()[pri:2, instance:listnerpractice.loginTest@491cc5c9]" name="Browser" duration-ms="4" started-at="2022-09-12T00:12:19 IST" finished-at="2022-09-12T00:12:19 IST">
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
at listnerpractice.loginTest.Browser(loginTest.java:45)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1259)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Browser -->
      </class> <!-- listnerpractice.loginTest -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
